exchanges/retry/src/retryExchange.ts

```diff
@@ -51,7 +51,7 @@ export const retryExchange = ({
       retry$,
       mergeMap((op: Operation) => {
         const { key, context } = op;
-        const retryCount = context.retryCount || 0;
+        const retryCount = context.retryCount++ || 1;
```

Comment:
This will mutate `context.retryCount` - I'd try and avoid this for operations

```suggestion
        const retryCount = context.retryCount++ || 1;
```

Reply:
Why is the mutation a bad thing? If we want to avoidit we would have to make it

```
 const retryCount = context.retryCount = context.retryCount + 1 || 1;
```